lib/pbench/server/api/__init__.py

```diff
@@ -82,13 +82,13 @@ def register_endpoints(api, app, config):
         IndexSearch, f"{base_uri}/index/search", resource_class_args=(config, logger),
     )
     api.add_resource(
-        IterationSampleNamespace,
-        f"{base_uri}/dataset/samples/namespace",
+        SampleNamespace,
+        f"{base_uri}/index/namespace/<string:target_document>",
         resource_class_args=(config, logger),
     )
     api.add_resource(
-        IterationSamplesRows,
-        f"{base_uri}/dataset/samples/rows",
+        SampleRows,
+        f"{base_uri}/index/rows/<string:target_document>",
```

Comment:
Yeah rows is very relational db term but I carried this term forward because we started the discussion on iteration samples API about naming them as `namespace` and `rows`. So thats where this is coming from. I agree we could convert it to `values` since its little more intuitive that rows if we all think that's better.

Reply:
At the time, we were looking at the Dashboard display which was quite definitely tabular, but that was when we were listing results, IIRC.  Now that we're talking about producing timeseries data, which will probably be plotted in a visualization, "rows" no longer seems appealing.

Reply:
Yes, a "namespace" query and a follow-on "values" or "data" query based on that namespace would work.